server/routes/strategies.ts

```diff
@@ -109,7 +109,7 @@ abstract class OAuthStrategy implements RegistrationStrategy {
 
 		let email: string = "";
 		if (profile.emails && profile.emails.length > 0) {
-			email = profile.emails[0].value;
+			email = profile.emails[0].value.trim();
```

Comment:
Is there any other possible canonicalization we need to do besides `.trim()`? Is there a library that will handle this for us? This doesn't seem that resilient - we could have other similar edge cases.

Reply:
Not really. This is mostly a hypothesis that emails with spaces returned in them might be caught by MongoDB at insert() but will be ignored by find(). Wasn't able to 100% reproduce though beyond a ton of looking at logs. Seems like this was mostly caused by the Georgia Tech CAS returning a username with a space in it for some reason.